docs/install/index.rst

```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
 
     gammapy info
 
-Congratulations! You are all set to start using Gammapy!
 
-If you're new to Python, ipython and Jupyter, read the :ref:`getting-started`
-guide. To learn how to use Gammapy, go to :ref:`tutorials`.
+Using other package managers
+----------------------------
+
+If you don't want to use Anaconda, you can use other package managers. To do so,the following pages contain detailed
+information about Gammapy dependencies and propose various installation options:
+
+* See gammapy install dependencies :ref:`install-dependencies`
+* If you want to install gammapy with pip see :ref:`install-pip`
+* If you want to use other package manager see :ref:`install-other`
 
-The following pages contain detailed information about Gammapy dependencies in
-various installation options:
 
-.. toctree::
-    :maxdepth: 1
+Download tutorials
+------------------
+
+You can now proceed to download the Gammapy tutorial notebooks and the example
+datasets used there (at the moment from CTA, H.E.S.S. and Fermi-LAT). The total
+size to download is about 100 MB.
+
+.. code-block:: bash
+
+    gammapy download tutorials --release 0.17
+    cd gammapy-tutorials
+    export GAMMAPY_DATA=$PWD/datasets
+
+You might want to put the definition of the ``$GAMMAPY_DATA`` environment
+variable in your shell profile setup file that is executed when you open a new
+terminal (for example ``$HOME/.bash_profile``).
+
+If you are not using the ``bash`` shell, handling of shell environment variables
+might be different, e.g. in some shells the command to use is ``set`` or something
+else instead of ``export``, and also the profile setup file will be different.
+
+On Windows, you should set the ``GAMMAPY_DATA`` environment variable in the
+"Environment Variables" settings dialog, as explained e.g.
+`here <https://docs.python.org/3/using/windows.html#excursus-setting-environment-variables>`__
+
+The datasets are curated and stable, the notebooks are still under development
+just like Gammapy itself, and thus stored in a sub-folder that contains the
+Gammapy version number.
+
+If there are issues, note that you can just delete the folder any time using ``rm
+-r gammapy-tutorials`` and start over.
+
+What next?
+----------
+
+Congratulations! You are all set to start using Gammapy!
 
-    dependencies
-    pip
-    other
+* If you're new to Python, ipython and Jupyter, read the :ref:`getting-started` guide.
```

Comment:
If you're new to conda, Python, ipython and Jupyter

Reply:
Done